tests/index.js

```diff
@@ -12,30 +12,30 @@ tape('Testing schema for POSTing ad slots', (t) => {
 	t.equals(Joi.validate(testData.slotWithEmptyDescription.marketAdd, schemas.adSlotPost).error, null, 'No error for slot with empty description field')
 
 	// TODO: Maybe stringify error object and look for error code as current check is too obscure
-	t.ok(Joi.validate(testData.slotWithInvalidType.marketAdd, schemas.adSlotPost).error instanceof Error, 'Error for slot with invalid type')
-	t.ok(Joi.validate(testData.slotWithBrokenDescription.marketAdd, schemas.adSlotPost).error instanceof Error, 'Error for slot with invalid description field')
-	t.ok(Joi.validate(testData.slotWithBrokenCreated.marketAdd, schemas.adSlotPost).error instanceof Error, 'Error for slot with invalid created timestamp')
-	t.ok(Joi.validate(testData.slotWithBrokenFallbackUnit.marketAdd, schemas.adSlotPost).error instanceof Error, 'Error for slot with invalid fallbackUnit field')
-	t.ok(Joi.validate(testData.slotWithBrokenTags.marketAdd, schemas.adSlotPost).error instanceof Error, 'Error for slot with broken tags field')
-	t.ok(Joi.validate(testData.slotWithBrokenTitle.marketAdd, schemas.adSlotPost).error instanceof Error, 'Error for slot with broken title field')
+	t.equals(Joi.validate(testData.slotWithInvalidType.marketAdd, schemas.adSlotPost).error.toString(), 'Error: TYPE_ERR_SLOT', 'Error for slot with invalid type')
```

Comment:
For the error checks have you tried to use ```t.throws```?
[](https://github.com/substack/tape#tthrowsfn-expected-msg)